rails/app/javascript/components/Map.jsx

```diff
@@ -87,65 +93,54 @@ export default class Map extends Component {
       } else {
         this.map.easeTo({ duration: 2000.0, ...frameOptions });
       }
-      return;
     }
   }
 
-  buildPopupHTML(marker) {
-    if (marker.properties.photo_url) {
-      if (marker.properties.region) {
-        if (marker.properties.type_of_place) {
-          return `<h1>${marker.properties.name}</h1>
-          <div class="ts-markerPopup-content">
-            <img src=${marker.properties.photo_url} />
-            <div>
-              <div>${I18n.t("region")}: ${marker.properties.region}</div>
-              <div>${I18n.t("place_type")}: ${
-            marker.properties.type_of_place
-          }</div>
-            </div>
-          </div>`;
-        } else {
-          return `<h1>${marker.properties.name}</h1>
-          <div class="ts-markerPopup-content">
-            <img src=${marker.properties.photo_url} />
-            <div>
-              <div>${I18n.t("region")}: ${marker.properties.region}</div>
```

Comment:
I'm a lil confused on how I18n.t was working in the JSX view previously. I removed it since it seemed it wasn't doing anything but I can put it back?

Reply:
It looks like it does do SOMETHING. I wonder if there is a hookin for I18n library in the JavaScript / asset pipeline.

<img width="222" alt="image" src="https://user-images.githubusercontent.com/2660801/101969939-58eebf80-3bf5-11eb-8d0a-f5f5b29adbb3.png">